.vitepress/config.ts

```diff
@@ -218,19 +218,9 @@ function sidebarHome() {
           text: "Quick start guide",
           link: "/tutorials/quick-start",
         },
+        { text: "Build a Rollup", link: "/tutorials/wordle" },
```

Comment:
```suggestion
        { text: "Build a rollup", link: "/tutorials/wordle" },
```

[nit] Not sure if we're sticking to sentence case for Rollkit docs or not

Reply:
personally I like capitals in menus, not sure if others have an opinion